client/question/domain/FeedbackObjectFactory.js

```diff
@@ -139,6 +139,17 @@ tie.factory('FeedbackObjectFactory', [
     };
 
     /**
+     * Appends a FeedbackParagraph of type output to the _paragraphs
+     * Array
```

Comment:
Minor nit, here and below: end these docstrings and arg descriptions with periods. (I think this was missed in some cases previously.)

Reply:
Yeah, it was copy/paste from existing code. Will add periods to the  existing code.